docs/rules/space-before-function-paren.md

```diff
@@ -26,303 +22,224 @@ This rule aims to enforce consistent spacing before function parentheses and as
 
 ## Options
 
-This rule takes one argument. If it is `"always"` then all named functions and anonymous functions must have space before function parentheses. If `"never"` then all named functions and anonymous functions must not have space before function parentheses. If you want different spacing for named and anonymous functions you can pass a configuration object as the rule argument to configure those separately (e. g. `{"anonymous": "always", "named": "never"}`). In this case, you can use "ignore" to only apply the rule to one type of function (e. g. `{"anonymous": "ignore", "named": "never"}` will warn on spaces for named functions, but will not warn on anonymous functions one way or the other).
+This rule can have either a simple string argument or an object literal (if more granular control is required).
+
+The string argument can either be:
+
+* `"always"` _(default)_ - all functions, named and anonymous, must have space before function parentheses.
+* `"never"` - all functions, named and anonymous, must not have space before function parentheses.
+
+If you want different spacing for named and anonymous functions you can pass a configuration object as the rule argument to configure those separately (e. g. `{"anonymous": "always", "named": "never"}`). In this case, you can use `"ignore"` to only apply the rule to one type of function (e. g. `{"anonymous": "ignore", "named": "never"}` will warn on spaces for named functions, but will not warn on anonymous functions one way or the other).
 
 The default configuration is `"always"`.
 
-### "always"
+## Examples
```

Comment:
Note: In the examples, I shrank functions from code like:

```js
function foo() {
  // ...
}
```

To use less space like:

```js
function foo() {}
```